third-party-programs.txt

```diff
@@ -0,0 +1,570 @@
+Unified Memory Framework (UMF) Third Party Programs File
```

Comment:
should we install this file (along with LICENSE)...?

Reply:
done